homeassistant/loader.py

```diff
@@ -97,10 +97,10 @@ class BlockedIntegration:
     "dreame_vacuum": BlockedIntegration(
         AwesomeVersion("1.0.4"), "crashes Home Assistant"
     ),
-    # Added in 2024.5.5 because of
-    # https://github.com/sh00t2kill/dolphin-robot/issues/185
+    # Added in 2024.8.0 because of
+    # https://github.com/sh00t2kill/dolphin-robot/issues/221
     "mydolphin_plus": BlockedIntegration(
-        AwesomeVersion("1.0.13"), "crashes Home Assistant"
+        AwesomeVersion("1.0.16"), "thousands of requests Maytronics login API per user"
```

Comment:
```suggestion
        AwesomeVersion("1.0.17"), "thousands of requests Maytronics login API per user"
```

Reply:
The version is the `lowest_good_version`

It looks like this issue was fixed in .17

Reply:
Yes it was fixed, but older versions of the integration abuse server with many login attempts instead of reusing token,
That's the reason I think we still need it (force users to upgrade),
If there is better way to handle it, pls let me know what i should do

Thanks